src/Examples/CoreExample/Views/Account/Connect.cshtml

```diff
@@ -28,138 +28,270 @@
         </div>
     </div>
     <div class="row">
-        <div class="col-md-4 col-sm-6 col-xs-12">
-            ID:
+        <div class="col-md-3 col-sm-6 col-xs-12">
```

Comment:
What's the purpose of aligning this one line like this? It only adds unnecessary spacing.

Reply:
Do you mean why we're not putting it in a table? 
It doesn't have sources & verifiers, so I didn't think it made sense to include it in the table. 

Or, if you meant why I've gone with a 3/9 split rather than 4/4 before, it's just because the Yoti ID is very long, this is so it fits on one line.